### PR TITLE
Update ML.NET tutorials to .NET 6

### LINKED ISSUE #26778: Update ML.NET API Tutorials & Code Samples to .NET 6 & VS2022
- Remove older versions of
- Update csproj in code samples to target .NET 6
- Use latest .NET templates for console & web apps (where appropriate)
- Update instructions to match .NET 6 & VS2022 conventions.

### PR DESCRIPTION
**Pull request by jwood803**
## Summary

Updates all of the ML.NET API tutorials to .NET 6.

Fixes #26778

Preview Links

File Type | File Name | Published Url
-- | -- | --
Content | docs/machine-learning/tutorials/github-issue-classification.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/image-classification.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/image-classification-api-transfer-learning.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification-api-transfer-learning?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/iris-clustering.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/iris-clustering?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/movie-recommendation.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommendation?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/object-detection-onnx.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-onnx?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/phone-calls-anomaly-detection.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/predict-prices.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/sales-anomaly-detection.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sales-anomaly-detection?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/text-classification-tf.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/text-classification-tf?branch=pr-en-us-27046
Content | docs/machine-learning/tutorials/time-series-demand-forecasting.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/time-series-demand-forecasting?branch=pr-en-us-27046

### CODE REVIEW

File: docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs [hunk 1]
```diff
@@ -1,191 +1,179 @@
...
+/// <summary>
+/// Class to hold the fixed length feature vector. Used to define the
+/// column names used as output from the custom mapping action,
+/// </summary>
+public class FixedLength
+{
+    /// <summary>
+    /// This is a fixed length vector designated by VectorType attribute.
+    /// </summary>
+    [VectorType(FeatureLength)]
```
Comment (jwood803) [inline docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs:176]: An error here is the only one with the snippets build: 
```
error CS8801: Cannot use local variable or local function 'FeatureLength' declared in a top-level statement in this context
```

I'm not quite sure how to fix this.

Comment (luisquintanilla) [reply docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs:176]: @jwood803 Not sure if this is the most elegant solution, but having any config / constant values in a static class works. I'll dig into it more.

```csharp
static class Config
{
    public const int FeatureLength = 600;
}

class FixedLength
{
    [VectorType(Config.FeatureLength)]
    public int[] Features { get; set; }
}
```

Comment (luisquintanilla) [reply docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs:176]: In the meantime, we can go with this solution though

File: docs/machine-learning/tutorials/image-classification-api-transfer-learning.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-- [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) or later with the ".NET Core cross-platform development" workload installed.
+- [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/).
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/image-classification-api-transfer-learning.md:28]: I think we still need to add the workload. Based on the .NET tutorial for console apps, it seems ".NET Desktop Development" workload is required.

https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio?pivots=dotnet-6-0

**Review by luisquintanilla — COMMENTED** [review]
@jwood803 Looks good to me. I added a comment re: workloads you'll need. Make sure to include that for each of the tutorial prereqs as part of the VS2022 requirement.

File: docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs (hunk 1 above)
Comment (jwood803) [reply docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs:176]: Thanks! Updated the snippet and added the info in the tutorial

File: docs/machine-learning/tutorials/image-classification-api-transfer-learning.md (hunk 2 above)
Comment (jwood803) [reply docs/machine-learning/tutorials/image-classification-api-transfer-learning.md:28]: They should all now be updated.

**Review by luisquintanilla — APPROVED** [review]
LGTM. Thanks @jwood803

**Pull request merged**